### DISCUSSION TITLE
Potential Video questions

### DISCUSSION THREAD

**Discussion by benngarnish** [jjburton/cgmTools#300] [Show and tell]
- [x] how to tag/place animation controllers into an existing build setup and how to structure the extra controllers. Where do they live within the existing build setup?
- [ ] hook a rig outside of MRS into the puppetNode meta rig system as if it was built as one. 
- [ ] Saving out an attribute map of the animation controllers. For example being able to save a rigging state of all controllers unlocks and unhidden. And another state that we can load that locks certain channels and hides them to so that animators aren't confused.

**Comment by jjburton** [comment]
### 11.14.2021

- Menagerie update
- New Neck/tail stuff
- New Tool
- Adding controls to a rig
--- Where do you put them?
